specification/apimanagement/resource-manager/Microsoft.ApiManagement/2017-03-01/apimapis.json

```diff
@@ -1728,6 +1954,63 @@
           "header": "customHeaderParameterName"
         }
       }
+    },
+    "SchemaCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SchemaContract"
+          },
+          "description": "Api Schema Contract value."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "The response of the list schema operation."
+    },
+    "SchemaContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SchemaContractProperties",
+          "description": "Properties of the Schema."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Schema Contract details."
+    },
+    "SchemaContractProperties": {
+      "properties": {
```

Comment:
provisioningState property missing? DO you need it? Just wanted to check since you are returning 201. But its not long running so its not required. But still checking.

Reply:
This is again a proxy resource in ARM. We dont need the provisioningState property.

---
In reply to: [143310129](https://github.com/Azure/azure-rest-api-specs/pull/1801#discussion_r143310129) [](ancestors = 143310129)

Reply:
Again, I think we should have a call and clarify a few things around tracked and non-tracked :). If it is long running operation, and you are returning 201 for the LRO, you will need to add a provisioning state property. Is it LRO?

Reply:
This is not a LRO.